.travis.yml

```diff
@@ -6,29 +6,38 @@ node_js:
 sudo: required
 dist: trusty
 
+cache:
+  yarn: true
+
 addons:
   apt:
     sources:
     - google-chrome
     packages:
     - google-chrome-stable
 
-cache:
-  directories:
-    - node_modules
+env:
+  - EMBER_TRY_SCENARIO=ember-1.13
+  - EMBER_TRY_SCENARIO=ember-lts-2.4
+  - EMBER_TRY_SCENARIO=ember-lts-2.8
+  - EMBER_TRY_SCENARIO=ember-release
+  - EMBER_TRY_SCENARIO=ember-beta
+
+matrix:
+  fast_finish: true
 
 before_install:
-  - "export DISPLAY=:99.0"
-  - "sh -e /etc/init.d/xvfb start"
-  - "npm install -g npm@^3"
-  - "npm config set progress false"
-  - "npm install phantomjs-prebuilt"
-  - "phantomjs --version"
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
+  - export PATH=$HOME/.yarn/bin:$PATH
+  - yarn global add phantomjs-prebuilt
+  - phantomjs --version
 
 install:
-  - npm install -g bower
-  - npm install
+  - yarn global add bower
+  - yarn install --no-lockfile
```

Comment:
@mmun I copied the ember-cli addon travis blueprint - https://github.com/ember-cli/ember-cli/blob/master/blueprints/addon/files/.travis.yml#L35